### PR TITLE
Add current_timestamp for Spark

### PR DESCRIPTION
**Pull request by Maayan-s**

### CODE REVIEW

**Comment by Maayan-s** [comment]
Following the latest comments in this issue:
https://github.com/elementary-data/elementary/issues/483

It's not official Spark support as we didn't add tests, but might help users and reduce the volume in #support.

File: macros/utils/cross_db_utils/current_timestamp.sql [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+
+{% macro default__current_timestamp() -%}
+    {% set macro = dbt.current_timestamp_backcompat or dbt_utils.current_timestamp %}
+    {% if not macro %}
+        {{ exceptions.raise_compiler_error("Did not find a `current_timestamp` macro.") }}
+    {% endif %}
+    {{ return(macro()) }}
+{%- endmacro %}
+
+{% macro databricks__current_timestamp() -%}
```
Comment (Maayan-s) [inline macros/utils/cross_db_utils/current_timestamp.sql:13]: TBH I don't understand why this works for Databricks. 
The adapter does not include a macro named `current_timestamp_backcompat`

File: macros/utils/cross_db_utils/timediff.sql [hunk 2]
```diff
@@ -3,19 +3,12 @@
     {{ return(adapter.dispatch('timediff', 'elementary')(timepart, first_timestamp, second_timestamp)) }}
 {%- endmacro %}
 
-{# Snowflake #}
+{# For Snowflake, Databricks, Redshift, Postgres & Spark #}
+{# the dbt adapter implementation supports both timestamp and dates #}
 {% macro default__timediff(timepart, first_timestamp, second_timestamp) %}
-    datediff({{ timepart }}, {{ first_timestamp }}, {{ second_timestamp }})
+    {{ elementary.datediff(first_timestamp, second_timestamp, timepart)}}
```
Comment (IDoneShaveIt) [inline macros/utils/cross_db_utils/timediff.sql:9]: Is it working for postgres?
It has different implementation from the rest (not sure if dbt.datediff) handles that for us or not so I am asking.

**Review by IDoneShaveIt — COMMENTED** [review]
Looking good!
Left one comment :)

File: macros/utils/cross_db_utils/timediff.sql (hunk 2 above)
Comment (Maayan-s) [reply macros/utils/cross_db_utils/timediff.sql:9]: Yes, checked and it does.

**Pull request merged**